addon/lib/survey.js

```diff
@@ -16,7 +16,7 @@ const querystring = require('sdk/querystring');
 const store = require('sdk/simple-storage').storage;
 
 const NUM_STARS = 5; // Number of survey stars
-const FIVE_MINUTES = 60 * 1000 * 5;
+const TEN_MINUTES = 60 * 1000 * 10;
```

Comment:
Nit: This should probably be named for purpose, not content - e.g. SURVEY_PROMPT_DELAY. Will probably merge anyway though